games/devtest/mods/testabms/README.md

```diff
@@ -0,0 +1,6 @@
+# Test ABMs
```

Comment:
please split the unit tests into a seperate commit (also so they're not squashed on merge)